### PR TITLE
11.0 mig stock pack operation auto fill

### PR DESCRIPTION
**Pull request by luistriana28**
@pedrobaeza Could you review this aportation?

Best Regards

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Hi, Luis, thanks for the contribution!

I'm not able to review all the PRs that incomes, but for sure if there's interest other will review it. You can also review other PRs and ask in exchange that they review yours.

File: stock_pack_operation_auto_fill/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+* The product has no tracking set (Technically `tracking` field should be equal to `none`).
+
+* The operation has not be processed (i.e `qty_done == 0`).
+
+* The operation has no package set (i.e `package_id` is empty).  
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/9.0
```
Comment (LoisRForgeFlow) [inline stock_pack_operation_auto_fill/README.rst]: 11.0

File: stock_pack_operation_auto_fill/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Stock Pack Operation Auto Fill',
+    'summary': """
+        Stock pack operation auto fill""",
+    'version': '10.0.1.0.1',
```
Comment (LoisRForgeFlow) [inline stock_pack_operation_auto_fill/__manifest__.py]: 11.0.1.0.0

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Thanks for the migration! See my comments inline plus:

* You should remove all `# -*- coding: utf-8 -*-` as it is not needed in python3.
* Squash together all transifex commits to reduce history lenght.

Always good to have a look to the [migration guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0)

File: stock_pack_operation_auto_fill/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import stock_picking
```
Comment (alan196) [inline stock_pack_operation_auto_fill/models/__init__.py]: `# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).`

File: stock_pack_operation_auto_fill/models/stock_picking.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
```
Comment (alan196) [inline stock_pack_operation_auto_fill/models/stock_picking.py]: `from odoo import _, api, fields, models`

File: stock_pack_operation_auto_fill/tests/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+from . import test_stock_picking
```
Comment (alan196) [inline stock_pack_operation_auto_fill/tests/__init__.py]: `# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).`

**Review by alan196 — CHANGES_REQUESTED** [review]
Runbot is in warning due the following message:
`The 'phonenumbers' Python module is not installed. Try: pip install phonenumbers.`

**Review by alan196 — APPROVED** [review]
👍 Technical review.
👍 Functional test.

**Comment by LoisRForgeFlow** [comment]
Functional review is fine :+1: , waiting for the [CLAs](https://github.com/OCA/stock-logistics-workflow/pull/399#issuecomment-358158599) before approving

**Comment by luistriana28** [comment]
@lreficent 

I already sent it! 

Regards!

**Comment by lasley** [comment]
ICLA for @luistriana28 confirmed

**Review by LoisRForgeFlow — APPROVED** [review]
Great, thanks for the migration!

**Review by ougc27 — APPROVED** [review]
:+1: Tested on runbot!

**Pull request merged**